software-stack/software-stack-q&a.md

```diff
@@ -63,7 +63,7 @@
 
 1. Care este asocierea apel de bibliotecă / apel de sistem?
 
-   (Nu înțeleg întrebarea)
+   Asocierea dintre un apel de bibliotecă și un apel de sistem este că un apel de bibliotecă este o formă de a realiza un apel de sistem.
```

Comment:
```suggestion
   Asocierea dintre un apel de bibliotecă și un apel de sistem este accea că un apel de bibliotecă poate rezulta într-un apel de sistem.
```

A libcall doesn't necessarily result in a syscall. Please reflect this in the answer like suggested.